src/ui/pages/WalletSelect/WalletSelect.tsx

```diff
@@ -100,6 +100,7 @@ export function WalletSelect() {
               setCurrentAddressMutation.mutate(wallet.address);
             }}
             selectedAddress={singleAddress}
+            showAddressValues={true}
```

Comment:
Maybe just `showAddresses`?

Reply:
I think this is more descriptive, my reasoning:
* `showAddress` is a single form, but this is a list
* `showAddresses` is less clear because "address" and "wallet" are often used interchangible, and this is already a wallet list. "Address value" seems to better convey that it means the raw `0x123...` value